solidity/dashboard/src/sagas/coverage-pool.js

```diff
@@ -399,6 +406,46 @@ export function* subscribeToWithdrawalCompletedEvent() {
   }
 }
 
+export function* subscribeToAuctionCreatedEvent() {
+  const requestChan = yield actionChannel(
+    RISK_MANAGER_AUCTION_CREATED_EVENT_EMITTED
+  )
+
+  while (true) {
+    yield take(requestChan)
+
+    const hasRiskManagerOpenAuctions = yield call(
+      Keep.coveragePoolV1.hasRiskManagerOpenAuctions
+    )
```

Comment:
I think we can skip this call because if an auction has been created we know that the risk manager has at least one open auction.

Reply:
27e566d66d3f2927b7ed6dc7672c137ba2db746e